python-backend/app/api/incidents/models/mine_incident.py

```diff
@@ -5,12 +5,14 @@
 from sqlalchemy.dialects.postgresql import UUID
 from sqlalchemy.schema import FetchedValue
 from app.extensions import db
-from ....utils.models_mixins import AuditMixin, Base
+
 from .mine_incident_followup_investigation_type import MineIncidentFollowupInvestigationType
-from app.api.mines.incidents.models.mine_incident_determination_type import MineIncidentDeterminationType
+
+from app.api.utils.models_mixins import AuditMixin, Base
+from app.api.incidents.models.mine_incident_determination_type import MineIncidentDeterminationType
```

Comment:
@Jsyro I remember you discussing that you thought incidents should be under mines, do you think this makes sense? We have some base level API methods so having 'incidients' at the top level makes sense to me, even if some routes are under /mines/.

Reply:
The discussion about asking about ALL incidents (across all mines), ended with us saying that `/mines/<guid>/incidents` is a specific mine and `/incidents` is all mines (instead of `/mines/incidents`. Mostly this came true because that results in both `/mines/<guid>/incidents/<guid>` vs `/mines/incidents/<guid>` being supported and we didn't want to implement both of those paths, or have two paths map to the same code.